src/lib/net/nettypes.h

```diff
@@ -0,0 +1,39 @@
+/* Copyright (c) 2003-2004, Roger Dingledine
+ * Copyright (c) 2004-2006, Roger Dingledine, Nick Mathewson.
+ * Copyright (c) 2007-2018, The Tor Project, Inc. */
+/* See LICENSE for licensing information */
+
+#ifndef TOR_NET_TYPES_H
+#define TOR_NET_TYPES_H
```

Comment:
There is a possible argument here that would argue about this file being called `types.h` since it is now in the "net" module by design.

Reply:
I'm trying hard to avoid having two headers with the same name (like net/types.h and int/types.h).